docs/user/unattended/storage.md

```diff
@@ -0,0 +1,956 @@
+the section "searching existing devices" of this document.
+
+## Entries for Describing the Devices
+
+The formal specification of the previous section can be extended as we dive into the structure.
+
+```
+Drive
+  search [<Search|string>]
+  alias [<string>]
+  encryption [<Encryption>]
+  filesystem [<Filesystem>]
+  ptableType [<string>]
+  partitions [<Partition[]>]
+
+VolumeGroup
+  search [<Search|string>]
+  alias [<string>]
+  name [<string>]
+  peSize [<number>]
+  physicalVolumes [<string[]>]
+  logicalVolumes [<LogicalVolume[]>]
+  delete [<boolean=false>]
+
+MdRaid
+  search [<Search|string>]
+  alias [<string>]
+  name <string>
+  level [<string>]
+  chunkSize [<number>]
+  devices [<string[]>]
+  size [<Size>]
+  encryption [<Encryption>]
+  filesystem [Filesystem]
+  ptableType [<string>]
+  partitions [<Partition[]>]
+  delete [<boolean=false>]
+
+BtrfsRaid
+  search [<Search|string>]
+  alias [<string>]
+  dataRaidLevel <string>
+  metadataRaidLevel <string>
+  devices [<string[]>]
+  label [<string>]
+  mkfsOptions [<string[]>]
+  [Btrfs]
+  delete [<boolean=false>]
+
+NFS
+  alias [<string>]
+  path [<string>]
+  mount [<MountAction>]
+
+Partition
+  search [<Search|string>]
+  alias [<string>]
+  id [<string>]
+  size [<Size>]
+  encryption [Encryption]
+  filesystem [<Filesystem>]
+  delete [<boolean=false>]
+  deleteIfNeeded [<boolean=false>]
+
+LogicalVolume
+  search [<Search|string>]
+  alias [<string>]
+  name [<string>]
+  size [<Size>]
+  pool [<boolean>]
+  usedPool [<string>]
+  stripes [<number>]
+  stripSize [<number>]
+  encryption [Encryption]
+  filesystem [<Filesystem>]
+  delete [<boolean=false>]
+  deleteIfNeeded [<boolean=false>]
+
+Encryption
+  reuse <Boolean>
+  type <EncryptionType>
+
+EncryptionType <EncryptionLUKS1|EncryptionLUKS2|EncryptionPervasiveLUKS2|"protected_swap"|"secure_swap"|"random_swap">
+
+EncryptionLUKS1
+  password <string>
+  keySize [<number>]
+  cipher [<string>]
+
+EncryptionLUKS2
+  password <string>
+  keySize [<number>]
+  cipher [<string>]
+  pdkdf [<string>]
+  label [<string>]
+
+EncryptionPervasiveLUKS2
+  password <string>
+
+Filesystem
+  reuse <Boolean>
+  type <string|Btrfs>
+  label [<string>]
+  mkfsOptions [<string[]>]
+  path <string>
+  mountOptions [<string[]>]
+  mountBy [<string>]
+
+Btrfs
+  subvolumePrefix [<string>]
+  subvolumes [<Subvolume[]>]
+  snapshots [<boolean=false>]
+  quotas [<boolean=false>]
+
+Size <string|SizeRange>
+
+SizeRange
+  min <string>
+  max <string>
+
+BootSettings
+  configure <boolean>
+  device <string>
+
+EncryptionSettings
+  method <string>
+  key [<string>]
+  pdkdf [<string>]
+  cipher [<string>]
+  keySize [<number>]
+```
+
+To illustrate how all that fits together, let's see the following example in which the first disk of
+the system is partitioned and a volume group is created on top of that partition (after encrypting
+it) to allocate two file systems.
+
+```json
+"storage": {
```

Comment:
Please, let's use the same indentation in all the examples (2 spaces).

Reply:
Fixed.